### DISCUSSION TITLE
Visibility duration of a flash

### DISCUSSION THREAD

**Discussion by mdeherder** [php-flasher/php-flasher#32] [Q&A]
Hi,
How to change the visibility time of a flash ?
(handler=template)

**Comment by yoeunes** [comment]
Hello Michel, I'm really sorry I didn't see your message.
I know it's too late, but you could change the duration with the `timeout` option.

**Comment by jonathanoeijoeng** [comment]
I'm using livewire and is it possible to change duration from 5s to 3s for all flasher?

Thanks,

> Hello Michel, I'm really sorry I didn't see your message.
> 
> I know it's too late, but you could change the duration with the `timeout` option.

**Reply by yoeunes** [reply]
hello @jonathanoeijoeng  you could try something like this : 

```php
<?php

return [
    'default' => 'flasher',

    'themes' => [
        'flasher' => [
            'options' => [
                'timeout' => 3000,
            ],
        ],
    ],
];
```

**Reply by jonathanoeijoeng** [reply]
Thanks for you reply.
Where i have to put this script? In flasher.php?

**Reply by yoeunes** [reply]
Yes in `config/flasher.php`, you need to create it manually

**Reply by jonathanoeijoeng** [reply]
it worked. Thanks so much
